core/server/master/src/main/java/alluxio/master/meta/UpdateCheck.java

```diff
@@ -98,7 +114,39 @@ public static String getUserAgentString(String clusterID) throws IOException {
         sysInfo = joiner.join(sysInfo, joiner.join(ec2Info));
       }
     }
-    return String.format("Alluxio/%s (%s)", ProjectConstants.VERSION, sysInfo);
+    return sysInfo;
+  }
+
+  /**
+   * Get the features information.
+   *
+   * @return a list of strings representing enabled features
+   */
+  @VisibleForTesting
+  public static List<String> getUserAgentFeatureString() {
```

Comment:
since this doesn't return a String anymore, rename to `getUserAgentFeatures()`?

Reply:
addressed